host/sgx/linux/sgxquoteprovider.c

```diff
@@ -141,6 +141,10 @@ oe_result_t oe_get_revocation_info(oe_get_revocation_info_args_t* args)
     uint8_t* p = 0;
     uint8_t* p_end = 0;
 
+#if defined(OE_USE_LIBSGX)
+    OE_CHECK(oe_initialize_quote_provider());
```

Comment:
Nice, we already have a function that limits doing this just once for any given process. That's really helpful